audit.go

```diff
@@ -0,0 +1,60 @@
+package authaus
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/IMQS/serviceauth"
+)
+
+type Auditor interface {
```

Comment:
This interface isn't actually used anywhere.
Create a concrete type that implements the Auditor interface, and talks to the IMQS auditor interface. The concrete type's name should reference the fact that it's the IMQS audit service, so something like ImqsAuditor